cli/valet.php

```diff
@@ -181,6 +181,35 @@
         info('The ['.$url.'] site will now serve traffic over HTTP.');
     })->descriptions('Stop serving the given domain over HTTPS and remove the trusted TLS certificate');
 
+    /**
+     * Create an Nginx proxy config for the specified domain
+     */
+    $app->command('proxy [domain] [host]', function ($domain, $host) {
```

Comment:
I think `domain` and `host` should probably be required. Not an expert on Silly, but it looks like dropping the `[]` should do it:

```
$app->command('proxy domain host', function ($domain, $host) {
```